src/BM2/DungeonBundle/Resources/translations/dungeons.en.yml

```diff
@@ -10,7 +10,9 @@ enter:
    and mortals are afraid of them, but young or desperate First Ones seek them for the experience and treasures they
    contain.<br />
    For sheer survival, you can join other First Ones and enter this dungeon together, fighting your way into its depths
-   until you beat it or it beats you. Teamwork is key.
+   until you beat it or it beats you. Teamwork is key. <br />
+   Though, if you feel particularly daring, you can brave the worst
+   this dungeon has to offer on your own.
```

Comment:
absolute no. I do not want dungeons to be solo-able. Never, ever. We can discuss lowering the minimum characters required to enter a dungeon, but I will not accept a pull request where it is one.